### PR TITLE
Tidy

### PR DESCRIPTION
**Pull request by apasel422**

### CODE REVIEW

**Comment by FlashCat** [comment]
r? @reem

(I've picked a reviewer for you, use r? to override)

**Comment by arthurprs** [comment]
Looks good to me.

File: src/lib.rs [hunk 1]
```diff
@@ -615,6 +614,22 @@ impl<K: Hash + Eq + PartialOrd, V: PartialOrd, S: HashState> PartialOrd for Link
...
+    fn lt(&self, other: &LinkedHashMap<K, V, S>) -> bool {
+        iter::order::lt(self.iter(), other.iter())
+    }
+
+    fn le(&self, other: &LinkedHashMap<K, V, S>) -> bool {
+        iter::order::le(self.iter(), other.iter())
+    }
+
+    fn ge(&self, other: &LinkedHashMap<K, V, S>) -> bool {
+        iter::order::ge(self.iter(), other.iter())
```
Comment (Gankro) [inline src/lib.rs]: It's not clear to me that ge/le/ne have different perf characteristics; surely they must all short-circuit at the same place?

Comment (apasel422) [inline src/lib.rs]: They have the same performance characteristics modulo the underlying `{eq, ne, lt, le, ge, gt}` impls. I think collection types should do their best to honor those (and other specializations).

**Comment by Gankro** [comment]
WFM

**Pull request merged**